### ISSUE TITLE
Would it be helpful in Speech Recognition especially in acoustic model to replace CTC with ACE?

### ISSUE THREAD

**Issue by nemoramo** [summerlvsong/Aggregation-Cross-Entropy#10]
Hello, I am writing this topic to ask if it will be useful in speech recognition tasks. I am going to test your ACE loss on my acoustic model. Hope it can produce comparable performance. I will show the result later.

**Comment by summerlvsong** [comment]
Yes, after feature extraction, there is no much difference between acoustic model and OCR model.
